tests/IAM/Domain/User/Model/EmailTest.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Twitter\Tests\IAM\Domain\User\Model;
+
+use PHPUnit\Framework\Attributes\CoversClass;
+use PHPUnit\Framework\Attributes\Group;
+use PHPUnit\Framework\Attributes\Test;
+use PHPUnit\Framework\TestCase;
+use Twitter\IAM\Domain\User\Model\Email;
+use Twitter\IAM\Domain\User\Model\Exception\InvalidEmailException;
+
+#[Group('unit')]
+#[CoversClass(Email::class)]
+class EmailTest extends TestCase
+{
+    #[Test]
+    public function canBeCreatedFromAValidEmailString(): void
```

Comment:
```suggestion
    public function canBeCreatedFromAValidEmail(): void
```
@adnlv (medium) To improve this test I would suggest creating a [Data Provider](https://docs.phpunit.de/en/12.5/attributes.html#dataprovider) with few correct variants.